tests/test_SequenceManager.cc

```diff
@@ -646,28 +651,20 @@ TEST_CASE("SequenceManager: git repository", "[SequenceManager]")
         SequenceManager manager{ git_dir };
 
         // find sequence unique ID
-        UniqueId uID{ 0 };
-        for (auto elm: manager.list_sequences())
-        {
-            if (elm.name == SequenceName{ "git_sequence_1" }) uID = elm.unique_id;
-        }
-        REQUIRE(uID == UniqueId{ 0xabcdef123456 });
+        auto s1 = find_sequence_by_name(manager.list_sequences(), "git_sequence_1");
+        REQUIRE(s1->unique_id == UniqueId{ 0xabcdef123456 });
```

Comment:
That is UB if the sequence is not found. Maybe we should add a `REQUIRE(s1.has_value())`.

Reply:
I had `REQUIRE(s)` before, but then, accessing a non value, should throw? No?
Checking and fixing if needed.